tests/gen_sim_input.py

```diff
@@ -48,6 +48,7 @@
                                 # The default is strings "1" ... "100"
 DIST_PARAM = None               # Parameter to pass to distribution
 NUM_CLIENTS = 100000            # Number of simulated clients
+VALUES_PER_CLIENT = 7
 
```

Comment:
If we're going to fix VALUES_PER_CLIENT, let's fix it to 1. In order to show the usefulness of -oneprr, we have to have a use case where a client generates lots of *identical* values to show that averaging attacks don't work. I didn't realize when originally working on this part but having >1 value per client is in no man's land in terms of understanding what's going on.

Either set it to exactly 1 to avoid any confusion or specially tailor it to return the same value (say) 20 times to show averaging attacks don't work. Thoughts?